safe-global/safe-wallet-web
# fix: do not offer executing unsigned txs with connected wallet

## What it solves
We currently offer executing unsigned txs through the connected wallet whilst having a subaccount opened.

## How this PR fixes it
Only adds the connected wallet as option if the transaction is already fully signed or the wallet is an owner.

## How to test it
1. Open a 1/n Subaccount whose parent is owned by the connected wallet
2. Create a new tx with the current Safes nonce
3. Select to Execute the tx directly
4. Observe that the connected wallet is not offered
5. Now only Sign the tx
6. When signed, click execute 
7. Observe that the connected wallet is now an option

## Checklist
